arm-timeseriesinsights/2017-02-28-preview/Swagger/timeseriesinsights.json

```diff
@@ -0,0 +1,1780 @@
+        "x-ms-examples": {
+          "EnvironmentsCreate": { "$ref": "../examples/EnvironmentsCreate.json" }
+        },
+        "description": "Create or update an environment in the specified subscription and resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "environmentName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$",
+            "minLength": 1,
+            "maxLength": 90,
+            "description": "Name of the environment"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/EnvironmentCreateOrUpdateParameters"
+            },
+            "description": "Parameters for creating an environment resource."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The existing environment definition was successfully updated.",
+            "schema": {
+              "$ref": "#/definitions/EnvironmentResource"
+            }
+          },
+          "201": {
+            "description": "The environment create request was accepted. Environment provisioning is an asynchroneous operation. You can periodically get your environment definition and monitor progress via the provisioningState property.",
+            "schema": {
+              "$ref": "#/definitions/EnvironmentResource"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "Environments"
+        ],
+        "operationId": "Environments_Get",
+        "x-ms-examples": {
+          "EnvironmentsGet": { "$ref": "../examples/EnvironmentsGet.json" }
+        },
+        "description": "Gets the environment with the specified name in the specified subscription and resource group.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "#/parameters/EnvironmentNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The environment definition was successfully retrieved and is in the response. If you are polling for the completion of a provisioning or scale operation, you can check its status via the provisioningState property.",
+            "schema": {
+              "$ref": "#/definitions/EnvironmentResource"
+            }
+          },
+          "404": {
```

Comment:
Thanks. I have added defaults to all operations. Generally 404, except for PUTs which are 400s.